release/build_release.sh

```diff
@@ -58,125 +58,170 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_BUILD_DIR}"
 }
 
-function main {
-	export ANT_OPTS="-Xmx10G"
+function execute_hotfix {
```

Comment:
@Wykenz, the methods `execute_hotfix` and `execute_release` seems to have many steps in common, in light of this can you try to optimize this code?

Additionally, can we adjust the methods' names to `build_release`, `build_hotfix` and `build_nightly` instead?

`build_*` seems to be more close to the purpose of the methods :slightly_smiling_face: